docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+    app:
+        build:
+            context: .
+            dockerfile: _docker/app/Dockerfile
+        volumes:
+            - ./:/var/www
+        container_name: todo_app
+        depends_on:
+            - db
+
+    nginx:
+        image: nginx:latest
+        volumes:
+            - ./:/var/www/
+            - ./_docker/nginx/conf.d/:/etc/nginx/conf.d/
+        ports:
+            - "8876:80"
+        container_name: todo_nginx
+        depends_on:
+            - app
+
+    db:
+        image: postgres:latest
+        restart: always
+        ports:
+            - 8889:3306
+        environment:
+            POSTGRES_DATABASE: '${DB_DATABASE}'
```

Comment:
taking these parameters from the application's .env is a bad way. For docker-compose, you must have a different configuration file or you must set the parameters right here.